nussl/ml/train/closures.py

```diff
@@ -213,6 +214,13 @@ def __init__(self, loss_dictionary, optimizer, model, *args, **kwargs):
         self.optimizer = optimizer
         self.model = model
 
+        # Save about training metadata to model.info
+        self.model.metadata['optimizer'] = {
+            'name': type(optimizer).__name__,
+            'params': optimizer.defaults  # All of the settings are stored here.
```

Comment:
Does this only give defaults for the optimizer or does it give what was actually used during training?

Reply:
Yeah, I tested it. It will reflect whatever values you pass into it. It's a confusingly named attribute, but I think the idea is that the params change throughout training so these are where they were initialized. ¯\_(ツ)_/¯